packages/plugin-electron-ipc/bugsnag-ipc-main.js

```diff
@@ -102,7 +102,7 @@ module.exports = class BugsnagIpcMain {
       }
       event.context = event.context || this.client._context
       event._metadata = { ...event._metadata, ...this.client._metadata }
-      event._features = { ...event._features, ...this.client._features }
+      event._features = [...event._features, ...this.client._features]
```

Comment:
In `client.js` there is `[...this._features, ...event._features]`. Should the order of the spread be the same here and there i.e. event first then client or client first then event?

Reply:
that's a really good question, actually. @gareththackeray would this kind of thing be defined in the spec?

Reply:
Any client features with a given name should be overridden by event features with the same name, and client features should be ordered first (as they are added before the event is created).

Reply:
should that be the same with all the other metadata, or just features?

Reply:
Yes you'd expect anything that was added to the event to override the client level stuff.

Reply:
so I should update this to perform a proper merge at this point? currently this could have duplicate flags.

Reply:
Oh, yes.